office-365-feed/operations.py

```diff
@@ -0,0 +1,275 @@
+"""
+  Copyright start
+  Copyright (C) 2008 - 2022 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end
+"""
+import requests
+import uuid
+import datetime
+import time
+import re
+from .constants import *
+from connectors.core.connector import get_logger, ConnectorError
+
+logger = get_logger('office-365-feed')
+
+
+def ip_to_indicator_type(ip):
```

Comment:
let this be a jinja in the create record playbook. we don't need type in action output
MS already classifies as IP and URL. we can just use that. further into ipv4 and ipv6 is not required